rocket_sqlite_oracle/src/main.rs

```diff
@@ -0,0 +1,251 @@
+use anyhow::{Context, Result};
+use rocket::fairing::AdHoc;
+use rocket::serde::json::Json;
+use rocket::serde::Serialize;
+use rocket::{fairing, get, routes, tokio, Build, Rocket, State};
+use rocket_db_pools::{sqlx, Connection, Database};
+use rocket_sqlite_oracle::kraken;
+use secp256k1::rand::thread_rng;
+use secp256k1::{PublicKey, SecretKey, Signature};
+use secp256k1_zkp::SECP256K1;
+use std::convert::{TryFrom, TryInto};
+use std::path::{Path, PathBuf};
+use structopt::StructOpt;
+use tokio::io::AsyncWriteExt;
+use url::Url;
+
+use baru::oracle::Message;
+
+#[derive(Database)]
+#[database("oracle_signatures")]
+struct ExampleDB(sqlx::SqlitePool);
```

Comment:
`Signatures` would be a more descriptive name.

Reply:
You can pack all the db stuff into a `db` module, then it reads like `db::Signatures`.